### PR TITLE
Adding data model details 

### PR DESCRIPTION
**Pull request by rajakvk**
Information collected from 
- "website-api-contract" repo
- swaggerDefinition.js
- "website-backend" repo model's folder code

### CODE REVIEW

File: users/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  'github_id': string,
+  'linkedin_id': string,
+  'twitter_id': string,
+  'instagram_id': string,
+  'website': string,
+  'github_display_name': string,
+  'isMember': boolean,
+  'userType': string,
+  'tokens': {},
+  'badges': []
```
Comment (swarajpure) [inline users/README.md:24]: Are you sure we're going to have badges here? In API contract for `badges` I saw that we're having a badges document and maintaining an array of users to which that particular badge has been assigned to.

Comment (swarajpure) [inline users/README.md:24]: Please add `status` as string here, which will contain the current status of user (ooo, active, idle)

File: auctions/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# Auctions
+
+## Firestore collections:
+
+- auctions
+- biddingLog
```
Comment (swarajpure) [inline auctions/README.md:6]: We're using:
`auctions`
`bids`

File: auctions/README.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  'end_time': <unix-timestamp>,
+  'bidders_and_bids': [
+    {
+      'bidder': <userId>,
+      'bid': <number>,
+      'timestamp': <unix-timestamp>
+    }
+  ],
+}
+```
```
Comment (swarajpure) [inline auctions/README.md:43]: This is how we're returning data and not how we're storing it in firestore

This is how we're storing data in `auctions` collection:
image

`bids` collection looks like this:
image

File: bids/README.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+Firestore Bids collection data model
+
+```
+{
+  auction_id: <auctionId>,
+  bidder: <bidder>,
+  bid: number,
+  time: timestamp
+}
+```
```
Comment (swarajpure) [inline bids/README.md:11]: This is perfect! ✅

**Review by swarajpure — CHANGES_REQUESTED** [review]
Providing data model for `auctions`

Also, requesting to add `status` in users data model and one doubt regarding `users` collection

Thank you for your hard work! ⚡ 🔥

File: auctions/README.md (hunk 2 above)
Comment (rajakvk) [reply auctions/README.md:6]: Updated: biddingLog => bids

File: users/README.md (hunk 1 above)
Comment (rajakvk) [reply users/README.md:24]: Added status: string

Comment (rajakvk) [reply users/README.md:24]: I copied structure from code base. Yes, we have separate badges collection in firestore. Not necessary here. Removing it.

File: auctions/README.md (hunk 3 above)
Comment (rajakvk) [reply auctions/README.md:43]: Updated structure based on screen shot above.

File: bids/README.md (hunk 4 above)
Comment (rajakvk) [reply bids/README.md:11]: Checking "discord webhook". ignore.

**Review by swarajpure — DISMISSED** [review]
Auctions, bids, notifications looks good to me!

File: stocks/README.md [hunk 5]
```diff
@@ -0,0 +1,4 @@
+# Stocks
+Firestore Stocks collection data model
+
+TBD
```
Comment (sumitd94) [inline stocks/README.md:4]:
```javascript
{
name: string,
price: number,
quantity: number
}

**Review by sumitd94 — DISMISSED** [review]
One small change for Stocks Model, the rest all looks good.

File: stocks/README.md (hunk 5 above)
Comment (rajakvk) [reply stocks/README.md]: Updated accordingly

**Review by swarajpure — APPROVED** [review]
Awesome job in here man! 🔥

**Review by harshith-venkatesh — APPROVED** [review]

**Review by sumitd94 — APPROVED** [review]

**Pull request merged**